models/dave_worlds/package.xml

```diff
@@ -3,7 +3,7 @@
   <name>dave_worlds</name>
   <version>3.1.1</version>
   <description>The dave_worlds package</description>
-
+  <author email="mabel@openrobotics.org">Mabel Zhang</author>
```

Comment:
I can't claim authorship here. I only added 1 file to the package very recently - probably why last modified shows me.

Reply:
Switched to @dtdavi1 with 7658a50